### ISSUE TITLE
make uninstall fails

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/lambdapi#615]
```
14:50 ~/src/lambdapi (master) make clean
make[1] : on entre dans le répertoire « /home/blanqui/src/lambdapi/editors/emacs »
find . -name '*.elc' -exec rm -f {} \;
make[1] : on quitte le répertoire « /home/blanqui/src/lambdapi/editors/emacs »
make[1] : on entre dans le répertoire « /home/blanqui/src/lambdapi/editors/vscode »
rm -rf node_modules/ out/
make[1] : on quitte le répertoire « /home/blanqui/src/lambdapi/editors/vscode »
14:51 ~/src/lambdapi (master) make
14:52 ~/src/lambdapi (master) make install
Error: The following <package>.install are missing:
- _build/default/editors/emacs/lambdapi-mode.install
Hint: try running: dune build @install
make: *** [Makefile:119 : install] Erreur 1
```
Note also that `make tests` run 105 tests only.

**Comment by fblanqui** [comment]
`dune build @install` works indeed.

On the other hand, `make uninstall` doesn't work, and the provided "Hint" is wrong:
```
14:56 ~/src/lambdapi (master) make uninstall
Error: The following <package>.install are missing:
- _build/default/editors/emacs/lambdapi-mode.install
Hint: try running: dune build @install
make: *** [Makefile:123 : uninstall] Erreur 1
```

**Comment by fblanqui** [comment]
105 is the number of lp files in tests/, but there are also 82 dk files.

**Comment by fblanqui** [comment]
Am I the only one to have this strange problem?

**Comment by gabrielhdt** [comment]
The hint works for me. In fact the uninstall step requires the `<package>.install` to know which file to remove.

**Comment by ejgallego** [comment]
Yup, in the past `dune install` used to build things automatically, but that was problematic for some reason so in the 2.0 version of the dune language it does require to build the pkg.install file first.

**Comment by gabrielhdt** [comment]
Is this fixed by #637 ?

**Comment by fblanqui** [comment]
Yes.
